File: CleanJavaRx/app/src/release/java/com/vascome/fogtail/utils/NoOpLeakCanaryProxy.java
package com.vascome.fogtail.utils;

import android.support.annotation.NonNull;

/**
 * Created by vasilypopov on 11/23/17
 * Copyright (c) 2017 MVPJava. All rights reserved.
 */

public class NoOpLeakCanaryProxy implements LeakCanaryProxy {
    @Override
    public void init() {
        // no-op.
    }

    @Override
    public void watch(@NonNull Object object) {
        // no-op.
    }
}

File: MosbyMVIKotlin/app/src/release/java/com/vascome/fogtail/utils/NoOpLeakCanaryProxy.kt
package com.vascome.fogtail.utils

/**
 * Created by vasilypopov on 11/23/17
 * Copyright (c) 2017 MVPJava. All rights reserved.
 */

class NoOpLeakCanaryProxy : LeakCanaryProxy {
    override fun init() {
        // no-op.
    }

    override fun watch(`object`: Any) {
        // no-op.
    }
}
